clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheStateServiceImpl.java

```diff
@@ -242,6 +250,139 @@ private ResourcePageSource createPageSource(String poolName, ServerSideConfigura
+      }
+    });
+
+    registerStatistic(store, storeName, "dataOccupiedMemory", STATISTICS_STORE_TAG, PROPERTY_STORE_KEY, new Callable<Number>() {
+      @Override
+      public Number call() throws Exception {
+        return store.dataAllocatedMemory();
+      }
+    });
+
+    registerStatistic(store, storeName, "occupiedMemory", STATISTICS_STORE_TAG, PROPERTY_STORE_KEY, new Callable<Number>() {
+      @Override
+      public Number call() throws Exception {
+        return store.occupiedMemory();
+      }
+    });
+
+    registerStatistic(store, storeName, "dataOccupiedMemory", STATISTICS_STORE_TAG, PROPERTY_STORE_KEY, new Callable<Number>() {
+      @Override
+      public Number call() throws Exception {
+        return store.dataOccupiedMemory();
+      }
+    });
+
+    registerStatistic(store, storeName, "entries", STATISTICS_STORE_TAG, PROPERTY_STORE_KEY, new Callable<Number>() {
+      @Override
+      public Number call() throws Exception {
+        return store.longSize();
+      }
+    });
+
+    registerStatistic(store, storeName, "usedSlotCount", STATISTICS_STORE_TAG, PROPERTY_STORE_KEY, new Callable<Number>() {
+      @Override
+      public Number call() throws Exception {
+        return store.usedSlotCount();
+      }
+    });
+
+    registerStatistic(store, storeName, "dataVitalMemory", STATISTICS_STORE_TAG, PROPERTY_STORE_KEY, new Callable<Number>() {
+      @Override
+      public Number call() throws Exception {
+        return store.dataVitalMemory();
+      }
+    });
+
+    registerStatistic(store, storeName, "vitalMemory", STATISTICS_STORE_TAG, PROPERTY_STORE_KEY, new Callable<Number>() {
+      @Override
+      public Number call() throws Exception {
+        return store.vitalMemory();
+      }
+    });
+
+    registerStatistic(store, storeName, "reprobeLength", STATISTICS_STORE_TAG, PROPERTY_STORE_KEY, new Callable<Number>() {
+      @Override
+      public Number call() throws Exception {
+        return store.reprobeLength();
+      }
+    });
+
+    registerStatistic(store, storeName, "removedSlotCount", STATISTICS_STORE_TAG, PROPERTY_STORE_KEY, new Callable<Number>() {
+      @Override
+      public Number call() throws Exception {
+        return store.removedSlotCount();
+      }
+    });
+
+    registerStatistic(store, storeName, "dataSize", STATISTICS_STORE_TAG, PROPERTY_STORE_KEY, new Callable<Number>() {
+      @Override
+      public Number call() throws Exception {
+        return store.dataSize();
+      }
+    });
+
+    registerStatistic(store, storeName, "tableCapacity", STATISTICS_STORE_TAG, PROPERTY_STORE_KEY, new Callable<Number>() {
+      @Override
+      public Number call() throws Exception {
+        return store.tableCapacity();
+      }
+    });
+
+  }
+
+  private void registerPoolStatistics(String poolName, ResourcePageSource pageSource) {
+    registerStatistic(pageSource, poolName, "allocatedMemory", STATISTICS_POOL_TAG, PROPERTY_POOL_KEY, new Callable<Number>() {
```

Comment:
`allocatedMemory` is perhaps better to be renamed to `allocatedSize` because if we look at the store stats, `size` means something different than `allocated`... Could you check the code to be sure we can call it allocatedMemory ?